include/dao/dao.js

```diff
@@ -710,6 +761,24 @@ module.exports = function DAOModule(pb) {
     };
 
     /**
+     * Gets all collection names
+     * @returns {DAO}
+     */
+    DAO.prototype.getAllCollections = function(cb) {
+        pb.dbm.getDb(this.dbName, function(err, db) {
+            if (util.isError(err)) {
+                return cb(err);
+            }
+            db.collections(function(err, items) {
```

Comment:
The new mongo driver no longer supports calling ```collections```. 
http://mongodb.github.io/node-mongodb-native/2.0/api/Db.html#listCollections

Is this function really needed?